### ISSUE TITLE
investspec "evapotranspiration" wording needs to be changed

### ISSUE THREAD

**Issue by newtpatrol** [natcap/invest.users-guide#146]
The Data Needs section of the User Guide for Urban Cooling and Annual Water Yield currently says this:

> Evapotranspiration ([raster](https://storage.googleapis.com/releases.naturalcapitalproject.org/invest-userguide/latest/en/input_types.html#raster), units: mm, required): Map of evapotranspiration values.

Just calling it "evapotranspiration" is not precise enough, because there is a difference between _potential_ ET and _reference_ ET, and both of these models' equations specify _reference_ ET (then they calculate potential ET using Kc provided in the biophysical table). So the Data Needs section should specify "reference evapotranspiration", like it does for Seasonal Water Yield. 

So please update this in investspec - thanks!

**Comment by newtpatrol** [comment]
I'm noticing that Jesse submitted a request for this specifically for Urban Cooling in March 2023, and it's still not updated. There are also a few other UG investspec updates that are pending from a while ago. Without intending to be whiny, this is a downside of relying on investspec for data documentation, such that Jesse and I can't update it immediately ourselves.

**Comment by davemfish** [comment]
> I'm noticing that Jesse submitted a request for this specifically for Urban Cooling in March 2023, and it's still not updated. There are also a few other UG investspec updates that are pending from a while ago. Without intending to be whiny, this is a downside of relying on investspec for data documentation, such that Jesse and I can't update it immediately ourselves.

Yeah, there's some room for improvement here. Thanks for bringing it up. A few things come to mind that we can do,
* Transfer the issue to invest, since that's where the change needs to be made (I will do this; github provides a "Transfer issue" button).
* Label the issue as a "bug" in order to catch our attention as higher priority. Anyone can do this using the "Labels" menu to the right ---->.
* @newtpatrol , @jagoldstein We can train you up to make these changes in invest by submitting a pull request. No obligation, it's always okay to create the issue and move on. But it's very much within reason for us to give you a quick tutorial for how to submit a PR for changes in wording to an invest model spec. Let me know if you're interested.
